unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Shortcuts/ShortcutsController.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using DCL;
+using UnityEngine;
+
+public class ShortcutsController : IDisposable
+{
+    internal InputAction_Trigger toggleQuestsPanel;
+    internal InputAction_Trigger toggleAvatarNames;
+    internal InputAction_Trigger toggleControls;
+    internal InputAction_Trigger toggleAvatarEditor;
+    internal InputAction_Trigger toggleExplore;
+    internal InputAction_Trigger toggleExpressionsHUD;
+    internal InputAction_Trigger toggleNavMap;
+
+    public ShortcutsController()
+    {
+        toggleQuestsPanel = Resources.Load<InputAction_Trigger>("ToggleQuestsPanelHUD");
+        toggleAvatarNames = Resources.Load<InputAction_Trigger>("ToggleAvatarNames");
+        toggleControls = Resources.Load<InputAction_Trigger>("ToggleControlsHud");
+        toggleAvatarEditor = Resources.Load<InputAction_Trigger>("ToggleAvatarEditorHud");
+        toggleExplore = Resources.Load<InputAction_Trigger>("ToggleExploreHud");
+        toggleExpressionsHUD = Resources.Load<InputAction_Trigger>("OpenExpressions");
+
+        //TODO: These require a small refactor to convert to DataStore
```

Comment:
Is not possible to address this small refactor on this PR?

Reply:
I didnt want to block the PR longer but I can tackle it down. I will start working on it but if I see that this PR is ready to merge, I will move the other one to a new PR.

Reply:
Done! @BrianAmadori would you mind taking a look?